tests/auth.setup.ts

```diff
@@ -9,25 +9,89 @@ import { authenticate, saveStorage } from '../playwright/helpers';
  *  - https://github.com/RedHatProductSecurity/osim/blob/main/src/stores/UserStore.ts
  */
 setup('authenticate', async ({ baseURL }) => {
-  const { refresh } = await authenticate();
+  const { refresh, cookies } = await authenticate();
+
+  // Determine environment - for dev environments, store refresh token in localStorage
+  // For non-dev environments, refresh tokens are handled via cookies
+  const isDev = process.env.OSIM_ENV === 'dev' || process.env.NODE_ENV === 'development';
```

Comment:
These are tests, is always a dev environment.

The setup is a "fixture" that we use to skip login during tests, there's no need to do checks, only save them